src/main/resources/theme-resources/messages/messages_fr.properties

```diff
@@ -1,27 +1,25 @@
-otpBodyHtml=<p>Quelqu'un a demandé un mot de passe à usage unique pour se connecter à {0}.</p><p><b>Code : {1}</b></p><p>Si vous n'avez pas demandé ce code, veuillez ignorer cet e-mail.</p>
+magicLinkBody=Quelqu''un a demandé un lien de connexion pour {0}.\n\nCliquez pour vous connecter.\n\n{1}\n\nSi vous n''avez pas demandé ce lien, veuillez ignorer cet e-mail.
+magicLinkBodyHtml=<p>Quelqu''un a demandé un lien de connexion pour {0}</p><p><a href="{1}">Cliquez pour vous connecter</a>.</p><p>Si vous n''avez pas demandé ce lien, veuillez ignorer cet e-mail.</p>
+otpSubject=Votre code d''accès pour {0}
+otpBody=Quelqu''un a demandé un mot de passe à usage unique pour se connecter à {0}.\n\nCode : {1}\n\nSi vous n''avez pas demandé ce code, veuillez ignorer cet e-mail.
+otpBodyHtml=<p>Quelqu''un a demandé un mot de passe à usage unique pour se connecter à {0}.</p><p><b>Code : {1}</b></p><p>Si vous n''avez pas demandé ce code, veuillez ignorer cet e-mail.</p>
 magicLinkContinuationSubject=Connexion à {0}
-magicLinkContinuationBody=Quelqu'un a demandé un lien de connexion pour {0}.\n\nCliquez pour vous connecter.\n\n{1}\n\nSi vous n'avez pas demandé ce lien, veuillez ignorer cet e-mail.
-magicLinkContinuationBodyHtml=<p>Quelqu'un a demandé un lien de connexion pour {0}</p><p><a href="{1}">Cliquez pour vous connecter</a>.</p><p>Si vous n'avez pas demandé ce lien, veuillez ignorer cet e-mail.</p>
+magicLinkContinuationBody=Quelqu''un a demandé un lien de connexion pour {0}.\n\nCliquez pour vous connecter.\n\n{1}\n\nSi vous n''avez pas demandé ce lien, veuillez ignorer cet e-mail.
+magicLinkContinuationBodyHtml=<p>Quelqu''un a demandé un lien de connexion pour {0}</p><p><a href="{1}">Cliquez pour vous connecter</a>.</p><p>Si vous n''avez pas demandé ce lien, veuillez ignorer cet e-mail.</p>
 
 # login
 magicLinkConfirmation=Vérifiez votre e-mail et cliquez sur le lien pour vous connecter !
 doResend=Renvoyer
 magicLinkContinuationConfirmation=Vérifiez votre e-mail et cliquez sur le lien pour vous connecter ! Veuillez ne pas fermer cet onglet.
-magicLinkSuccessfulLogin=Session d'authentification confirmée. Veuillez revenir à l'onglet de la page de connexion.
+magicLinkSuccessfulLogin=Session d''authentification confirmée. Veuillez revenir à l''onglet de la page de connexion.
 magicLinkFailLogin=La session d'authentification a expiré. Veuillez fermer cet onglet et redémarrer le processus de connexion.
 loginPage=Page de connexion
 multipleSessionsError=Plusieurs sessions de connexion ouvertes sur le même navigateur. Veuillez les fermer et redémarrer la connexion.
 
 # admin text
 ext-magic-form-display-name=Lien magique
 ext-magic-form-help-text=Connectez-vous avec un lien magique qui sera envoyé à votre e-mail.
-magic-link-continuation-form-display-name=Lien magique
-magic-link-continuation-form-help-text=Connectez-vous avec un lien magique qui sera envoyé à votre e-mail.
```

Comment:
I believe these need to be preserved.

Reply:
Sorry for this one, I added the lost lines back.